Products/DataCollector/zenmodeler.py

```diff
@@ -1110,10 +1115,10 @@ def mainLoop(self, driver):
         if self.clients:
             self.log.error("Modeling cycle taking too long")
             return
-
+        self.didCollect = False
```

Comment:
the flag needs to be cleared every time a modeling run starts. It may be that the assignment should be removed from the `ZenModeler.__init__()`